contracts/sidechain/bridge/BridgeDelegateSender.sol

```diff
@@ -8,5 +8,28 @@ import { Ownable } from "@openzeppelin/contracts-0.8/access/Ownable.sol";
  * @dev Sends tokens to L1 via a bridge
  */
 abstract contract BridgeDelegateSender is Ownable {
-    function send(address _to, uint256 _amount) external virtual;
+    /* -------------------------------------------------------------------
+       Storage 
+    ------------------------------------------------------------------- */
+    address public crv;
+
+    address public l1Receiver;
+
+    address public l2Coordinator;
+
+    /* -------------------------------------------------------------------
+       Setter Functions
+    ------------------------------------------------------------------- */
+
+    function setL1Receiver(address _l1Receiver) external onlyOwner {
+        require(_l1Receiver != address(0), "!0");
+        l1Receiver = _l1Receiver;
+    }
+
+    function setL2Coordinator(address _l2Coordinator) external onlyOwner {
+        require(_l2Coordinator != address(0), "!0");
+        l2Coordinator = _l2Coordinator;
+    }
+
+    function send(address _to, uint256 _amount) external virtual onlyOwner {}
```

Comment:
This can just be `function send(address _to, uint256 _amount) external virtual` to make sure it doesn't get left as an empty block I think.